Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -25,9 +25,7 @@ class HistoryPanel: SiteTableViewController, HomePanel {
     private var sectionOffsets = [Int: Int]()
 
     override func reloadData() {
-        let opts = QueryOptions()
-        opts.sort = .LastVisit
-        profile.history.get(opts).uponQueue(dispatch_get_main_queue()) { result in
+        profile.history.getSitesByLastVisit(100).uponQueue(dispatch_get_main_queue()) { result in
```

Comment:
Can you file a bug to make history dynamically fetch more data as you scroll? I don't really mind a limit on TopSites, but having one on History feels kinda bad to me.